src/php/bin/php_extension_doxygen_filter.awk

```diff
@@ -12,38 +12,63 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+function sed_gensub(regexp, replacement, how, target,     cmd_, ret_) {
```

Comment:
There's a bit of unnecessary space. Please remove it?

Reply:
the spaces in the parameter list are necessary to separate out local variables